pkg/statements/sql.go

```diff
@@ -91,6 +91,21 @@ WHERE
   table_name = $2`
 
 	// ---------------------------------------------------------------------------
+	PG_MATERIALIZED_VIEW_SCHEMA = `
+SELECT 
+  attname as column_name, 
+  atttypid::regtype AS data_type,
+  (case when attnotnull IS TRUE then 'NO' else 'YES' end) as is_nullable,
+  null as character_maximum_length,
+  null as character_set_catalog,
+  null as column_default
+FROM
+  pg_attribute
+WHERE
+  attrelid = $1::regclass AND attnum > 0
+  AND NOT attisdropped;`
+
```

Comment:
@sosedoff You're talking about `statements.PG_OBJECTS` query, yep, that works. But I'm talking about next step - `statements.PG_TABLE_SCHEMA`, which is querying `information_schema`, where is no info about materialized views.

Do you have materialized views working now?